faq/tidb.md

```diff
@@ -882,7 +882,7 @@ See [Syncer User Guide](/reference/tools/syncer.md).
 
 ##### How to configure to monitor Syncer status?
 
-Download and import [Syncer Json](https://github.com/pingcap/docs/blob/master/dev/etc/Syncer.json) to Grafana. Edit the Prometheus configuration file and add the following content:
+Download and import [Syncer Json](https://github.com/pingcap/docs/blob/master/etc/Syncer.json) to Grafana. Edit the Prometheus configuration file and add the following content:
```

Comment:
P.S. This is a JSON file, so we can link to Github and users can download this file from there.

Reply:
Got it.